src/dotnet/AzureAI.Proxy/Models/AzureOpenAIResponses/ChatCompletion.json

```diff
@@ -0,0 +1,62 @@
+{
```

Comment:
What are these files? Do we need example JSON files in the implementation?

Reply:
this is for future reference - building classes based on API version for cleaner serialization of responses. 
leave them, there is no impact

Reply:
removed them....have another idea - commit is coming